pokemongo_bot/cell_workers/transfer_pokemon.py

```diff
@@ -7,7 +7,24 @@
 class TransferPokemon(BaseTask):
     SUPPORTED_TASK_API_VERSION = 1
 
+    def initialize(self):
+        self.min_empty_space = self.config.get('min_empty_space', None)
+
+    def get_pokemon_slot_left(self):
+        return self.bot._player["max_pokemon_storage"] - self.bot.get_inventory_count('pokemon')
+
     def work(self):
+        if self.min_empty_space is not None:
+            free_poke_space = self.get_pokemon_slot_left()
+            if free_poke_space >= self.min_empty_space:
+                print free_poke_space
+                self.emit_event(
```

Comment:
I imagine this is going to log **all** the time. Do we really need an event for this? @douglascamata What do you think?